galasa-simbank-tests/dev.galasa.simbank.tests/src/main/java/dev/galasa/simbank/tests/ProvisionedAccountCreditTests.java

```diff
@@ -58,6 +56,21 @@ public class ProvisionedAccountCreditTests {
      * credit the account, then checks the balance again. The test passes if the
      * final balance is equal to the old balance + the credited amount.
      * 
+     * This test is an improved version of BasicAccountCreditTest.
+     * Log in is not hard coded and makes use of the credentials.properties file to get the credentials.
```

Comment:
It also uses the manager to store the login process and credential handling. I think thats worth mentioning